x-pack/plugins/transform/public/app/sections/transform_management/components/edit_transform_flyout/edit_transform_flyout_callout.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiTextColor,
+} from '@elastic/eui';
+
+import { useDocumentationLinks } from '../../../../hooks/use_documentation_links';
+export const EditTransformFlyoutCallout: FC = () => {
+  const { esTransformUpdate } = useDocumentationLinks();
+
+  return (
+    <EuiCallOut>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiIcon type="help" />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          {' '}
+          <EuiTextColor color="subdued">
+            {i18n.translate('xpack.transform.transformList.editFlyoutCalloutText', {
+              defaultMessage:
+                'This form allows you to update an existing transform. The list of properties that you can update is a subset of the list that you can define when you create a transform.',
+            })}
+          </EuiTextColor>
+          <EuiLink href={esTransformUpdate} target="_BLANK">
```

Comment:
I wonder if the link to the docs might look better in the top right of the flyout? Labelled just as `Transform docs` as you have on the parent list page.

Reply:
I took that design from EUI docs (`Flyout with banner`): 

![image](https://user-images.githubusercontent.com/230104/81074919-f78da200-8ee9-11ea-84c8-9afee5cf7c38.png)

Suggest to leave it as is in this PR, but happy to discuss alternatives.

Reply:
OK! Maybe just change the link to the docs to `View docs` from `Update transforms docs` ?